front/src/applications/stdcm/components/StdcmResults/SendToRailwayManagerModal.tsx

```diff
@@ -0,0 +1,416 @@
+import { useMemo, useState, useRef, useCallback } from 'react';
+
+import { Button, Input, Select, TextArea } from '@osrd-project/ui-core';
+import { Download, CheckCircle } from '@osrd-project/ui-icons';
+import { PDFDownloadLink } from '@react-pdf/renderer';
+import dayjs from 'dayjs';
+import utc from 'dayjs/plugin/utc';
+import { useTranslation } from 'react-i18next';
+
+import {
+  type StdcmSimulationInputs,
+  type StdcmSuccessResponse,
+  type SimilarTrainWithSecondaryCode,
+  type StdcmResultsOperationalPoint,
+} from 'applications/stdcm/types';
+import {
+  osrdRailwayManagerApi,
+  type BodyLastMinuteRequest,
+} from 'common/api/osrdRailwayManagerApi';
+import MAIN_API from 'config/config';
+import {
+  createLinkedTrainPayload,
+  createSimilarTrainPayload,
+  getStopDurationTime,
+  transformStepsToApiFormat,
+  validateTolerance,
+} from 'modules/SimulationReportSheet/utils/formatSimulationReportSheet';
+import type { StdcmPathStep } from 'reducers/osrdconf/types';
+import type { DeploymentSettings } from 'utils/hooks/useDeploymentSettings';
+import { kmhToMs, tToKg } from 'utils/physics';
+
+import StdcmSimulationReportSheet from './StdcmSimulationReportSheet';
+
+dayjs.extend(utc);
+
+type SendToRailwayManagerModalProps = {
+  onClose: () => void;
+  consist: StdcmSimulationInputs['consist'];
+  stdcmData: StdcmSuccessResponse;
+  linkedTrains: StdcmSimulationInputs['linkedTrains'];
+  simulationReportSheetNumber: string;
+  operationalPointsList: StdcmResultsOperationalPoint[];
+  simulationSheetLogo?: string;
+  similarTrains: SimilarTrainWithSecondaryCode[];
+  deploymentSettings?: DeploymentSettings;
+};
+
+type Option = { value: string; label: string };
+
+const SendToRailwayManagerModal = ({
+  consist,
+  onClose,
+  stdcmData,
+  linkedTrains,
+  simulationReportSheetNumber,
+  operationalPointsList,
+  simulationSheetLogo,
+  similarTrains,
+  deploymentSettings,
+}: SendToRailwayManagerModalProps) => {
+  const { t } = useTranslation('stdcm', { keyPrefix: 'simulation.results' });
+
+  const steps = stdcmData?.simulationPathSteps ?? [];
+
+  const firstStep = steps[0] as Extract<StdcmPathStep, { isVia: false }>;
+  const lastStep = steps[steps.length - 1] as Extract<StdcmPathStep, { isVia: false }>;
+
+  const scheduledDepartureTime = dayjs(stdcmData.departure_time);
+  const departureTime = dayjs(firstStep.arrival);
```

Comment:
Why not use `Date` ?
```suggestion
  const scheduledDepartureTime = new Date(stdcmData.departure_time);
  const departureTime = new Date(firstStep.arrival);
```

Reply:
Done